net/base/tracing.h

```diff
@@ -8,7 +8,7 @@
 #include "build/build_config.h"
 #include "net/base/cronet_buildflags.h"
 
-#if BUILDFLAG(CRONET_BUILD) && !BUILDFLAG(IS_APPLE) || 1
+#if BUILDFLAG(CRONET_BUILD) && !BUILDFLAG(IS_APPLE) || defined(STARBOARD)
```

Comment:
Probably change this to COBALT_PENDING_CLEAN_UP as we will want to use base/trace_event once that's properly implemented using perfetto